### ISSUE TITLE
X584

### ISSUE THREAD

**Issue by SeaMan75** [kodemeister/X584#45]
Сейчас делаю студенту программу в X584. Готов написать нормальную программу за умеренное вознаграждение. Потому что та версия скорее всего делалась бесплатно и без мотивации сделать нормально. Поиска нету, контекстного меню нету. Просто ввести микрокод нельзя, надо его искать в дереве. Сотню строк кода придется набирать часа три! Неужели в вузе не учат программировать...

**Comment by kodemeister** [comment]
> Потому что та версия скорее всего делалась бесплатно

Верно. X584 - это студенческий хобби-проект, который мы написали just for fun, когда сами учились на третьем курсе. Какой-либо оплаты разумеется никогда не планировалось и не планируется.

> Готов написать нормальную программу за умеренное вознаграждение.

Здесь ничем помочь не могу :man_shrugging: Я давно никак не связан с вузом.

> Поиска нету, контекстного меню нету. Просто ввести микрокод нельзя, надо его искать в дереве.

С удовольствием приму пулл реквесты, добавляющие недостающие возможности. Код открыт под GPLv3, к разработке приглашаются все желающие :smile:

**Comment by danil-kondr2016** [comment]
Я так понимаю, вам нужен ассемблер для X584, потому что в GUI делать ввод микрокода не совсем рационально.

**Comment by danil-kondr2016** [comment]
Репозиторий x584asm перенесён на Gitflic:
https://gitflic.ru/project/danila-kondr/x584asm
